schemas/hash.yml

```
@@ -5,12 +5,16 @@
   type: group
   short: Hashes, usually file hashes.
   description: >
-    The hash fields represent different hash algorithms and their values.
+    The hash fields represent different bitwise hash algorithms and their values.
 
     Field names for common hashes (e.g. MD5, SHA1) are predefined. Add fields for other hashes
     by lowercasing the hash algorithm name and using underscore separators as appropriate
     (snake case, e.g. sha3_512).
 
+    Note that this fieldset should be used for common hashes that may be computed
```

```suggestion
    Note that this fieldset is used for common hashes that are computed
```

nit: minor rewording suggestion.

I believe we can say authoritatively all fields here are computed in this way? Or are there exceptions?

---

All fields currently are computed that way, and this wording is to add that `hash` should stay that way, whereas any other "hash" that is based off of specific attributes of an object (like imphash) should go into a different set of fields.